frontend/src/components/TodoTask/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,35 @@
+import { Button } from 'antd';
+import { StarFilled, StarOutlined, BorderOutlined, CheckSquareOutlined } from '@ant-design/icons';
+import { blue } from '@ant-design/colors';
+
+export type IconButtonProps = {
+  onClick(): void;
+  color?: string;
+  variant: 'star' | 'check';
+  state: boolean;
```

Comment:
Nie do końca rozumiem nazwę tego propsa

Reply:
Jeśli to boolean to wykorzystaj też proszę składnię "isPies"

Reply:
isChecked ?

Reply:
done